src/mbgl/gl/context.cpp

```diff
@@ -456,13 +456,17 @@ Framebuffer Context::createFramebuffer(const Texture& color) {
 
 Framebuffer
 Context::createFramebuffer(const Texture& color,
-                           const Renderbuffer<RenderbufferType::DepthComponent>& depthTarget) {
+                           const Renderbuffer<RenderbufferType::DepthComponent>& depth) {
+    if (color.size != depth.size) {
+        throw std::runtime_error("Renderbuffer size mismatch");
+    }
     auto fbo = createFramebuffer();
     bindFramebuffer = fbo;
-    MBGL_CHECK_ERROR(glFramebufferTexture2D(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0, GL_TEXTURE_2D, color.texture, 0));
-    MBGL_CHECK_ERROR(glFramebufferRenderbuffer(GL_FRAMEBUFFER, GL_DEPTH_ATTACHMENT, GL_RENDERBUFFER, depthTarget.renderbuffer));
+    MBGL_CHECK_ERROR(glFramebufferTexture2D(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0, GL_TEXTURE_2D,
+                                            color.texture, 0));
+    MBGL_CHECK_ERROR(glFramebufferRenderbuffer(GL_FRAMEBUFFER, GL_DEPTH_ATTACHMENT, GL_RENDERBUFFER, depth.renderbuffer));
     checkFramebuffer();
-    return { depthTarget.size, std::move(fbo) };
+    return { color.size, std::move(fbo) };
```

Comment:
Only formatting/variable names changed here (except for size equality check); can we avoid this change please?